### ISSUE TITLE
remove methods in Command interface, which do not belong there

### ISSUE THREAD

**Issue by sadovnikov** [ContainerSolutions/minimesos#225]
Interface `com.containersol.minimesos.main.Command` contains methods, which are not applicable for majority of command. Therefore these methods have to be removed

**Comment by sadovnikov** [comment]
@alex-glv During the work on #220 we had to delete "hooks". It happened because their purpose was not clear, they were not documented and did not contain any references to issues in GH. Can we chat about them and possibly recreate in minimesos

**Comment by alex-glv** [comment]
The idea of hooks was to execute post-<command> actions.
If you start a cluster, a printServiceUrl hook would print all services that are started after command UP has been executed.

Now, when I start the cluster, the info is not printed anymore:
```
alg@alnix ~/Sites/minimesos % bin/minimesos up --consul
Pulling images required to launch minimesos ...
Finished pulling images.
Cluster 2819919326 - start
Image [containersol/consul-server:0.6] not found. Pulling...
alg@alnix ~/Sites/minimesos % 
```

**Comment by sadovnikov** [comment]
I logged #236 to restore and to complete introduction of consul

**Comment by sadovnikov** [comment]
This ticket is resolved by #233
